ROADMAP.md

```diff
@@ -90,16 +90,57 @@ See details in [GitHub milestone v1.3.0](https://github.com/ratify-project/ratif
 
 See details in [GitHub milestone v1.4.0](https://github.com/ratify-project/ratify/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.4.0).
 
-### v2.x
+### v2.0.0-alpha.1
+**Status**: In procoess
+**Target date**: May 15, 2025
+
+**Major features**
+
+- Create and implement ratify core library([ratify-go](https://github.com/ratify-project/ratify-go)), which should be in v1.0.0-alpha.1.
+- Implement new `Store` interface in [ratify-go](https://github.com/ratify-project/ratify-go).
+- Implement new `PolicyEnforcer` interface in [ratify-go](https://github.com/ratify-project/ratify-go).
+- Implement notation-verifier under [ratify-verifier-go](https://github.com/ratify-project/ratify-verifier-go) which should be in v1.0.0-alpha.1.
+- Create v2 branch in [ratify](https://github.com/ratify-project/ratify) repo and implement the new ratify add-on integrating with new `ratify-go` and `ratify-verifier-go` libraries.
```

Comment:
I feel this is the implementation's Pov. I would expect we architect the roadmap from the user's perspective, i.e. what features will be delivered to users to support which scenario. Maybe I can revise it with my my understanding.

Reply:
good call, I'll rephrase it later.

Reply:
Just updated